mindee/parsing/common/product.py

```diff
@@ -0,0 +1,13 @@
+from typing import Optional
+
+class Product:
+    """Class for keeping track of a product's info."""
+    name: Optional[str]
+    version: Optional[str]
+
+    def __init__(self, name=None, version=None) -> None:
```

Comment:
pass in the json and assign from there rather than having the fields passed directly

```suggestion
    def __init__(self, raw_prediction: StringDict) -> None:
```